src/docfx/Errors.cs

```diff
@@ -341,6 +341,16 @@ public static Error RedirectionUrlNotFound(string from, SourceInfo<string> sourc
 
             public static Error CircularRedirection(SourceInfo? source, IEnumerable<FilePath> redirectionChain)
                 => new Error(ErrorLevel.Warning, "circular-redirection", $"Build has identified circular redirection: {string.Join(" --> ", redirectionChain)}.", source);
+
+            /// <summary>
+            /// If the source-path defined in .openpublishing.redirection.json cannot be found,
+            /// should give this warning to users.
+            /// </summary>
+            /// Behavior: ✔️ Message: ✔️
```

Comment:
message is not confirmed with PM yet

Reply:
Fixed.

Reply:
Offline synced with @yufeih, we can use `missing-attribute` for this usage
https://github.com/dotnet/docfx/blob/c9aaefec21a15af46c024e2421c7cbbb83724586/src/docfx/Errors.cs#L563

Reply:
Done